specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
You can just say `string` for now.  We should start to use the `Time` type from the PVS prelude.